webapp/src/locales/en_US.ts

```diff
@@ -0,0 +1,106 @@
+import { LocaleType } from "./index";
+
+const en_US: LocaleType = {
+  Error: {
+    Unauthorized: "Currently not logged in.",
+    MustAuthorized: "Currently not logged in, please log in and use.",
+    Network: "The network has wandered off, please try again later!",
+    LocalCache: "Oops, something went wrong!",
+    LocalCacheTry: "Please clean the cache and try again!",
+    LocalCacheClean: "Clear All Cache",
+  },
+  Dashboard: {
+    QPH: "QPH",
+    QPS: "Avg. QPS",
+    QPM: "Avg. QPM",
+    Backends: "Backends",
+    BackendsOffline: "Backends Offline",
+    BackendsOnline: "Backends Online",
+    StartTime:"Started at",
+    Summary: "Summary",
+    QueryDistribution: "Query Distribution/hour.",
+    QueryCount: "Query Count",
+  },
+  Menu: {
+    Header: {
+      PersonalCenter: "Profile",
+      Logout: "Logout"
+    },
+    Sider: {
+      Dashboard: "Dashboard",
+      Cluster: "Cluster",
+      History: "History",
+      ResourceGroup: "Resource Group",
+      Selector: "Selector",
+    }
+  },
+  Auth: {
+    LoginTitle: "Welcome",
+    tips: {
+      tip1: "Sign in",
+      tip2: " Trino Gateway ",
+      tip3: "Account"
+    },
+    RegisterTitle: "Sign up account",
+    Tips: "The first login will generate a new account after verification",
```

Comment:
What do you mean by generating a new account?

Reply:
Sorry, I copied it and forgot to delete it